traffic_ops/traffic_ops_golang/auth.go

```diff
@@ -20,16 +20,18 @@ package main
  */
 
 import (
+	"context"
 	"database/sql"
-
 	"github.com/apache/incubator-trafficcontrol/lib/go-log"
-
 	"github.com/jmoiron/sqlx"
 )
 
 // PrivLevelInvalid - The Default Priv level
 const PrivLevelInvalid = -1
 
+// this level will turnoff authentication on an endpoint
+const Unauthenticated = 0
```

Comment:
This makes me anxious. It'd be easy to accidentally make a route unauthenticated, _especially_ since 0 is the default value of numeric types. It'd be better if we could figure out a way to indicate in the Routing table itself that a route is intended to be unauthenticated.

Reply:
I agree, will refactor to include an authenticated boolean on a route that defaults to true.